modules/billing.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * service_description/rhacs-cloud-service-service-description.adoc
+:_content-type: CONCEPT
+[id="billing_{context}"]
+= Billing
+
+Customers can purchase a {product-title-managed-short} subscription on the Amazon Web Services (AWS) marketplace.
+The service cost is charged hourly per secured core, or vCPU of a node belonging to a secured cluster.
+
+.Example
```

Comment:
This is showing up  as "Example 1. Example" in the preview. Maybe change title to "Subscription example"?